### ISSUE TITLE
ListIteratorNoNulls has a typo

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos#1479]
In `ListIteratorNoNull` has a typo:
```
    @Override
    public T previous() {
        final T next = this.listiterator.previous();
        if (next == null) {
            throw new IllegalStateException("Previous item is NULL");
        }
        return next;
    }
```
should be:
```
    @Override
    public T previous() {
        final T prev = this.listiterator.previous();
        if (prev == null) {
            throw new IllegalStateException("Previous item is NULL");
        }
        return prev;
    }
```

**Comment by victornoel** [comment]
@fabriciofx someone fixed it in #1483 :)

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1479` is not assigned, can't get performer
